database/revert/V20170114_2116__createAttendeeRSVP.revert.sql

```diff
@@ -1 +1,5 @@
-DROP TABLE `attendee_rsvps`;
\ No newline at end of file
+DROP TABLE `attendee_rsvps`;
+DELETE FROM `mailing_lists` WHERE `name`='attendees';
```

Comment:
I just realized that for the RSVP migration it adds attendees and deletes those three so I thought the revert for that migration should delete attendees and add the three that the migration deletes.